src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Value/EnumConverter.cs

```diff
@@ -378,7 +378,7 @@ private static bool TryParseEnumCore(ref Utf8JsonReader reader, JsonSerializerOp
             return success;
         }
 #else
-        private static bool TryParseEnumCore(string? enumString, JsonSerializerOptions options, out T value)
+        private static bool TryParseEnumCore(string? enumString, JsonSerializerOptions _, out T value)
```

Comment:
```suggestion
        private static bool TryParseEnumCore(string? enumString, JsonSerializerOptions options, out T value)
```
This looks like unintentional change. The signature of the two method variants should be in sync.

Reply:
Right, I did `git restore` using current main as source so that's why it ended up here.